SpecFlowProject1/StepDefinitions/LanguageActionsStepDefinitions.cs

```diff
@@ -30,8 +30,15 @@ public void WhenUserClicksOnAddNewButtonUnderLanguages()
             actions.AddLanguages(webdriver);
             
         }
+        [When(@"user enters language andproficiency level '([^']*)'")]
+        public void WhenUserEntersLanguageAndproficiencyLevel(string code)
+        {
+
+            actions.addLanguageandProficiency(webdriver, code);
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
please make use of Hooks , you might need to create the Common driver object there!
Please see
https://codebun.com/how-to-implement-hooks-in-specflow-c/